<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentMessageManagementSystemClass</class>
 <widget class="QMainWindow" name="StudentMessageManagementSystemClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>学生通讯录管理系统 (Student Contact Management System)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="prevPageButton">
      <property name="text">
       <string>上一页</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="pageLabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="nextPageButton">
      <property name="text">
       <string>下一页</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="displayArea">
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="actionInsert"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menuQuery">
    <property name="title">
     <string>查询(&amp;Q)</string>
    </property>
    <addaction name="actionQueryByID"/>
    <addaction name="actionQueryByName"/>
    <addaction name="actionQueryYoungest"/>
    <addaction name="actionQueryByCoordX"/>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>显示(&amp;D)</string>
    </property>
    <addaction name="actionDisplayPreorder"/>
    <addaction name="actionDisplayInorder"/>
    <addaction name="actionDisplayPostorder"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuQuery"/>
   <addaction name="menuDisplay"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>从文件读取(&amp;O)</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>写入文件(&amp;S)</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
  </action>
  <action name="actionInsert">
   <property name="text">
    <string>插入学生信息(&amp;I)</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>删除学生信息(&amp;D)</string>
   </property>
  </action>
  <action name="actionQueryByID">
   <property name="text">
    <string>按学号查询...</string>
   </property>
  </action>
  <action name="actionQueryByName">
   <property name="text">
    <string>按姓名查询...</string>
   </property>
  </action>
  <action name="actionQueryYoungest">
   <property name="text">
    <string>查询年龄最小的学生</string>
   </property>
  </action>
  <action name="actionQueryByCoordX">
   <property name="text">
    <string>按地址横坐标查询...</string>
   </property>
  </action>
  <action name="actionDisplayPreorder">
   <property name="text">
    <string>前序遍历显示</string>
   </property>
  </action>
  <action name="actionDisplayInorder">
   <property name="text">
    <string>中序遍历显示</string>
   </property>
  </action>
  <action name="actionDisplayPostorder">
   <property name="text">
    <string>后序遍历显示</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于(&amp;A)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
